.gitignore

```diff
@@ -0,0 +1,2 @@
+# Build folder is generated by kas tool
+build
```

Comment:
Please, add additional folders

Reply:
Fixed and added the modules which are handled by kas so that they're ignored by git.